src/make/Processor.cpp

```diff
@@ -739,26 +742,36 @@ Processor::_BuildCommand(data::RuleActionsCall* actionsCall)
 	// and "<" and ">")
 	data::VariableDomain builtInVariables;
 
+	// TODO: Support RuleActions::UPDATED
+	// TODO: Support RuleActions::EXISTING
+	auto setBoundPaths =
+		[this](StringList& boundPaths, const data::TargetList& targets) {
```

Comment:
I would suggest to name the function `setBoundTargets` and also the parameters `boundTargets`. Otherwise it's confusing why it's "paths" here and "targets" where it is used.

Reply:
Sure; I was worried about it being confused with the separate `boundTargets` variable, but I think changing it is still less confusing.

Do you also think the use of lambda here is acceptable? I thought it wasn't useful to make this another private function since it's only used here, but it might be more readable to extract it.

Reply:
> Do you also think the use of lambda here is acceptable? I thought it wasn't useful to make this another private function since it's only used here, but it might be more readable to extract it.

Personally, I think it's fine. When I learned C++, this was not available, but I figured out what this is and thought it was a great application of DRY without adding so much overhead.